lc0/src/utils/transpose.cc

```diff
@@ -26,7 +26,7 @@ void TransposeTensor(const std::vector<int>& dims, std::vector<int> order,
   }
   std::vector<int> cur_idx(dims.size());
   for (int _ = 0; _ < from.size(); ++_) {
-    size_t from_idx = 0;
+    std::size_t from_idx = 0;
```

Comment:
It's better to replace it with `int` then.
Seems like it's the only place in the code where I decided to use size_t, everywhere else int is used.